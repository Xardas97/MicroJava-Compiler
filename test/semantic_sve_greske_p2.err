ERROR 22:46:55,989 - Linija 3: Ulazna metoda 'main' ne sme imati povratnu vrednost
ERROR 22:46:55,989 - Linija 3: Ulazna metoda 'main' ne sme imati argumente
ERROR 22:46:55,990 - Parsiranje NIJE uspesno zavrseno!
