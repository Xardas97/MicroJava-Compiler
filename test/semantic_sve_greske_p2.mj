program semantic_sve_grekse_p2
{
    int main(int c) {
        return 1;
    }
}