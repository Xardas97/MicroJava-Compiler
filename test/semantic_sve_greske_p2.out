INFO  22:46:55,987 - Linija 3: Nadjeno Meth main: Int, 0, 0
INFO  22:46:55,987 - Linija 3: Nadjeno Var c: Int, 0, 1
INFO  22:46:55,990 - ==============SADRZAJ TABELE SIMBOLA===================
INFO  22:46:55,990 - 
Type int: int, -1, -1 
Type char: char, -1, -1 
Con eol: char, 10, 0 
Con null: Class [], 0, 0 
Meth chr: char, 0, 1 
   Var i: int, 0, 1 

Meth ord: int, 0, 1 
   Var ch: char, 0, 1 

Meth len: int, 0, 1 
   Var arr: Arr of notype, 0, 1 

Type bool: bool, -1, -1 
Prog semantic_sve_grekse_p2: notype, 0, 1 
   Meth main: int, 0, 1 
      Var c: int, 0, 1 



